plugins/inputs/http_response/README.md

```diff
@@ -24,6 +24,10 @@ This input plugin will test HTTP/HTTPS connections.
   # {'fake':'data'}
   # '''
 
+  # Log network errors in the Telegraf log
+  # Turned off by default to avoid filling the logs with exessive repetitive strings
+  # log_network_errors = false
```

Comment:
Instead of this option log the network issues at debug level with `log.Printf("D! foo")`